### PR TITLE
feat: update `binaryTargets` reference for Linux Alpine arm64

### PR DESCRIPTION
**Pull request by jkomyno**
## Changes

- update `binaryTargets`'s reference to reflect the newest Linux Alpine arm64 support
- specify that Prisma supports `x86_64` architectures by default

## What issue does this fix?

Contributes to https://github.com/prisma/prisma/issues/8478

### CODE REVIEW

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 1]
```diff
@@ -1185,12 +1193,12 @@ Defines a single-field ID on the model.
...
-| `clustered` | **No**   | `Boolean` | Defines whether the ID is clustered or non-clustered. Defaults to `true`. <br /><br />SQL Server only. In preview in versions 3.13.0 and later, and in general availability in versions 4.0.0 and later.                                                       |
+| Name        | Required | Type      | Description                                                                                                                                                                                                                                   |
+| ----------- | -------- | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| `map`       | **No**   | `String`  |                                                                                                                                                                                                                                               |
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:1198]: Accidental removal?

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 2]
```diff
@@ -1185,12 +1193,12 @@ Defines a single-field ID on the model.
...
+| `length`    | **No**   | `number`  | Allows you to specify a maximum length for the subpart of the value to be indexed.<br /><br />MySQL only. In preview in versions 3.5.0 and later, and in general availability in versions 4.0.0 and later.                                    |
+| `sort`      | **No**   | `String`  | Allows you to specify in what order the entries of the ID are stored in the database. The available options are `Asc` and `Desc`.<br /><br />In preview in versions 3.5.0 and later, and in general availability in versions 4.0.0 and later. |
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:1200]: Also accidental removal?

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 3]
```diff
@@ -1464,14 +1472,14 @@ Defines a multi-field ID (composite ID) on the model.
...
+| `map`       | **No**   | `String`           | The name of the underlying primary key constraint in the database                                                                                                                                                                             |
+| `length`    | **No**   | `number`           | Allows you to specify a maximum length for the subpart of the value to be indexed.<br /><br />MySQL only. In preview in versions 3.5.0 and later, and in general availability in versions 4.0.0 and later.                                    |
+| `sort`      | **No**   | `String`           | Allows you to specify in what order the entries of the ID are stored in the database. The available options are `Asc` and `Desc`.<br /><br />In preview in versions 3.5.0 and later, and in general availability in versions 4.0.0 and later. |
+| `clustered` | **No**   | `Boolean`          | Defines whether the ID is clustered or non-clustered. Defaults to `true`.<br /><br />SQL Server only. In preview in versions 3.13.0 and later, and in general availability in versions 4.0.0 and later.                                       |     |
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:1482]: Same here.

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 4]
```diff
@@ -2605,14 +2613,11 @@ Defines meta information about the relation. [Learn more](/concepts/components/p
...
+| `name`       | `String`           | Sometimes (e.g. to disambiguate a relation)                                                                                       | Defines the name of the relationship. In an m-n-relation, it also determines the name of the underlying relation table.            | `"CategoryOnPost"`, `"MyRelation"`                    |
+| `fields`     | `FieldReference[]` | On [annotated](/concepts/components/prisma-schema/relations#annotated-relation-fields-and-relation-scalar-fields) relation fields | A list of [fields](/concepts/components/prisma-schema/data-model#defining-fields) of the _current_ model                           | `["authorId"]`, `["authorFirstName, authorLastName"]` |
+| `references` | `FieldReference[]` | On [annotated](/concepts/components/prisma-schema/relations#annotated-relation-fields-and-relation-scalar-fields) relation fields | A list of [fields](/concepts/components/prisma-schema/data-model#defining-fields) of the model on _the other side of the relation_ | `["id"]`, `["firstName, lastName"]`                   |
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:2620]: Same here.

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 5]
```diff
@@ -619,7 +627,7 @@ True or false value.
 | ----------- | --------------- |
 | PostgreSQL  | `bigint`        |
 | SQL Server  | `int`           |
-| MySQL       | `BIGINT`        |
+| MySQL       | `INT`           |
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:630]: Accidental?

**Review by janpio — APPROVED** [review]
Actual `binaryTargets` look good.

**Review by janpio — CHANGES_REQUESTED** [review]
Unrelated changes seem problematic.

**Comment by jkomyno** [comment]
> Unrelated changes seem problematic.

That looks like some troubles when using Github's UI for editing the docs. Will clone the repo locally and push a new PR.

**Comment by jkomyno** [comment]
Deprecated by https://github.com/prisma/docs/pull/4491

**Pull request closed without merging**